Makefile

```diff
@@ -42,6 +50,25 @@ test-ci-unit: test-internal
 	@which goveralls > /dev/null || go get -u -f github.com/mattn/goveralls
 	goveralls -coverprofile=$(coverfile) -service=travis-ci || echo -e "\x1b[31mCoveralls failed\x1b[m"
 
+install-vendor: .gitmodules
```

Comment:
ah yeah, forgot about the node module.

I'm fine either way. Someone can probably spend a day rewriting the license module in go so we don't end up having a node module in go repos :).